src/Widgets/TopNavigationWidget/SubComponents/LanguageSwitch.tsx

```diff
@@ -41,7 +41,7 @@ export const LanguageSwitch = connect(function LanguageSwitch({
   return (
     <InPlaceEditingOff>
       <NavDropdown
-        title={<LanguageLabel root={activeSite} className="d-none" />}
+        title={<LanguageLabel root={activeSite} textClassName="d-none" />}
```

Comment:
* [ ] FYI The only values are `"d-none"` and `undefined`. For more clarity we could make this a boolean, e.g., `showLabel`.

Reply:
I will address this in a follow up.

Reply:
Addressed in https://github.com/Scrivito/scrivito-portal-app/pull/809.